tcms/testplans/static/testplans/js/get.js

```diff
@@ -162,6 +161,20 @@ function attachEvents(testPlanId, permissions) {
         });
     }
 
+    // get details and draw expand area only on expand
+    $('#confirmed-testcases .list-group-item').click(function(ev) {
```

Comment:
Add a `js-testcase-row` or similar class on the div. There could be other list-group-item elements if we change the layout and it will fuck up this selector. Also don't limit by confirmed or not b/c unconfirmed cases will still have the same functionality.